Begin.py/begin.py

```diff
@@ -52,76 +49,85 @@ def __repr__(self):
         s += 'MIJN EIGENSCHAP:\n' + str(self.my_feature)
         return s
 
-
     # Voeg hier andere methodes toe.
     # Je hebt in het bijzonder methodes nodig die het model vullen.
 
     # METHODS
-    def read_text_from_file(self, filename):
+
+    @staticmethod
+    def read_text_from_file(filename):
```

Comment:
zoals gemeld in een andere comment heb ik self.text weggehaald. Hierdoor heeft read_text_from_file geen dependencies op "state" (in dit geval geen dependecy meer op een instantie van de klasse, ookwel: "self") en kon deze methode static gemaakt worden. (static geeft aan dat een methode niet afhankelijk is van state)

Static methodes lijken me iets waar jullie nog niet van hoeven te weten maar even een best practice erin gooien leek me niet kwaad kunnen.